styles.css

```diff
@@ -0,0 +1,513 @@
+body {
```

Comment:
Your page currently has a scroll effect which provides bad user experience. It also make your items appear unaligned. 

https://user-images.githubusercontent.com/48546992/212504417-b6c67793-ed61-49b1-9946-e0a7ab24a8f4.mov


I suggest you fix this by first:
use the universal css selector `* ` and give your whole page 0 margin and padding like this
`*{margin: 0;
padding : 0;
box-sizing: border-box
width: 100%}`

This will improve user experience significantly.

Reply:
done